library/ajax/i18n_generator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * i18n_generator script
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Amiel Elboim <amielel@matrix.co.il>
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2019 Amiel Elboim <amielel@matrix.co.il>
+ * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+require_once(dirname(__FILE__) . "/../../interface/globals.php");
+
+$sql = "SELECT c.constant_name, d.definition FROM lang_definitions as d
+        JOIN lang_constants AS c ON d.cons_id = c.cons_id
+        WHERE d.lang_id = ?";
+$tarns = sqlStatement($sql, $_GET['lang_id']);
+$json = array();
+while ($row = SqlFetchArray($tarns)) {
+    $json[$row['constant_name']] = $row['definition'];
+}
+echo json_encode($json, JSON_UNESCAPED_UNICODE);
```

Comment:
Does this page actually output JSON? If so, we should probably update the header to `application/json` so the browser knows it's JSON

Reply:
I think es fetch is default json